src/express/controllers/api/action.ts

```diff
@@ -0,0 +1,110 @@
+import Review, {findCodeReviewRecord} from '../../../service/Review';
+import type {ChatPostMessageArguments} from '@slack/web-api';
+import type {RequestHandler} from 'express';
+import {formatApiCodeReviewRecord} from './reviews';
+import {postSlackMessage} from '../../../bolt/utils';
+
+export const actionController: RequestHandler = (req, res) => {
+    const action = req.params.action;
+    const value = req.params.value;
+
+    if (!req.user) {
+        res.json({});
+        return;
+    }
+
+    const {sid: slackUserId, fn: displayName} = req.user as {sid: string; fn: string};
+
+    if (!slackUserId) {
+        res.json({
+            error: 'Not authorized',
+        });
+        return;
+    }
+
+    if (['approve', 'claim', 'close', 'request-change', 'remove'].includes(action)) {
+        void findCodeReviewRecord({id: parseInt(value)}).then((codeReview) => {
```

Comment:
I have never seen void used like this before 😮 is it declaring a return type?

Reply:
Due to strict mode, eslint expecting full chain.then().catch() call.  Yes, it's a void execution trick conversion because the calling function is not async and we can't call await.